server/chat-commands/info.ts

```diff
@@ -2616,7 +2616,7 @@ export const commands: ChatCommands = {
 
 		if (this.broadcasting) {
 			if (isPMOrPersonalRoom) {
-				target = Chat.filter(this, target, user, room, connection, this.pmTarget)!;
+				target = Chat.filter(this, target)!;
```

Comment:
I saw a `context.filter` function defined. Can it be used in these places? Just `target = this.filter(target)!;`?

Reply:
Yeah, it can.